src/api/Nhs.Appointments.Core/AvailabilityService.cs

```diff
@@ -55,6 +55,17 @@ public async Task SetAvailabilityAsync(DateOnly date, string site, Session[] ses
         await availabilityStore.ApplyAvailabilityTemplate(site, date, sessions, mode);
     }
 
+    public async Task<IEnumerable<DailyAvailability>> GetDailyAvailability(string site, DateOnly from, DateOnly to)
+    {
+        if (string.IsNullOrEmpty(site))
+            throw new ArgumentException("Site must have a value.");
+
+        if (from > to)
+            throw new ArgumentException("From date must be before to date.");
```

Comment:
Isn't the first case covered in the request validator? 
The 2nd case isn't, but I think it could with a `.When()` or similar?

Reply:
They are, I forgot to remove them once I'd added in the validator, will remove them.